src/server/game/Globals/ObjectMgr.h

```diff
@@ -624,6 +624,16 @@ struct CharacterTemplate
 
 typedef std::unordered_map<uint32, CharacterTemplate> CharacterTemplateContainer;
 
+struct SceneTemplate
+{
+    uint32 sceneId;
```

Comment:
Public class/struct members should be named with UpperCamelCase style (default access level in a struct is public)